src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -58,10 +58,7 @@ private Object readResolve() {
 
     private SealightsCLIBuildStep resolveTechOption(){
         CommandMode.ConfigView config = (CommandMode.ConfigView) commandMode;
-        String packagesIncluded = config.getPackagesIncluded();
-        String packagesExcluded = config.getPackagesExcluded();
-        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = new DescribableList<>(JavaOptions.DescriptorImpl.NOOP);
-        technologyOptions.add(new JavaOptions(packagesIncluded,packagesExcluded));
+        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = config.getTechOptions();
```

Comment:
Seems like you have changed the logic. Am I missing something? Let's talk in the office.